export.py

```diff
@@ -1779,6 +1779,21 @@ def has_emissive_material(db):
     return False
 
 
+def export_for_bake(db):
+    export = True
+    node_names = []
+    for mat in db.material:
+        if mat and mat.node_tree:
+            nt = mat.node_tree
+            for n in nt.nodes:
+                node_names.append(n.name)
+    if node_names:
+        if 'PxrBakeTexture' not in node_names:
+            export = False
+
```

Comment:
Especially since that function has to return a result based on the names of all the nodes in the material.  If it runs the first node and it's not a bake texture that would return False and end the routine right?